solutions/observability/cicd.md

```diff
@@ -18,14 +20,18 @@ To provide monitoring dashboards, alerting, and root cause analysis on pipelines
 
 ## CI/CD observability architectures [ci-cd-architecture]
 
-Using the APM Server, connect all your OpenTelemetry native CI/CD tools directly to Elastic {{observability}}.
+Using the {{apm-server}}, connect all your OpenTelemetry native CI/CD tools directly to Elastic {{observability}}.
+
+:::{tip}
+We recommend using the [{{edot}} Collector](opentelemetry://reference/index.md) as the preferred way to collect CI/CD observability data.
```

Comment:
```suggestion
We recommend using the [{{edot}} Collector](opentelemetry://reference/index.md) to collect CI/CD observability data.
```
This makes it sound like you can set a "preferred" and "non-preferred" way, like perhaps the non-preferred is a fallback method. I don't know enough to say whether that's the case, but assuming it's not, I think this is more clear.